apprise/plugins/NotifyStreamlabs.py

```diff
@@ -0,0 +1,278 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 <example@example.com>
+# All rights reserved.
+#
+# This code is licensed under the MIT License.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files(the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions :
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+# For this to work correctly you need to register an app
+# and generate an access token
+#
+#
+#  This plugin will simply work using the url of:
+#     streamlabs://access_token/
+#
+# API Documentation on Webhooks:
+#    - https://dev.streamlabs.com/
+#
+import requests
+
+from .NotifyBase import NotifyBase
+from ..common import NotifyType
+from ..utils import validate_regex
+from ..AppriseLocale import gettext_lazy as _
+
+
+class NotifyStreamlabs(NotifyBase):
+    """
+    A wrapper to Streamlabs Donation Notifications
+
+    """
+    # The default descriptive name associated with the Notification
+    service_name = 'Streamlabs'
+
+    # The services URL
+    service_url = 'https://streamlabs.com/'
+
+    # The default secure protocol
+    secure_protocol = 'strmlabs'
+
+    # A URL that takes you to the setup/help of the specific protocol
+    setup_url = 'https://github.com/caronc/apprise/wiki/Notify_streamlabs'
+
+    # Streamlabs Api endpoint
+    notify_url = 'https://streamlabs.com/api/v1.0/donations'
```

Comment:
Just curious why your using apprise to generate donations, but i do see that streamlabs has a `alert` option.  Would the service be better off to support that? I was jsut curious how you hoped to leverage your great work here?  People who use the Apprise/Streamlabs can only use it to donate to a creator? Thoughts?

Reply:
I had though about having a param to specify alert or donation, but that can be added later if someone has a need for it.

apprise is a dependency within this project:
https://github.com/apprises/apprise-transactions

A demo can be seen here: https://www.reddit.com/r/Monero/comments/l4ab2s/transparent_monero_donations_demo/

Reply:
I read through the documentation for alerts.
I agree that alerts should be used here and not donations.
I will adapt the code, thanks for pointing this out.

Reply:
Sure, i mean i love the idea of setting a mode from `donation` and `alert` (defaulting to `alert`) too. That's  fantastic idea.  Don't worry or feel the need to do it in this PR, but definitely an awesome thing to consider down the road! :+1:

Reply:
Ok, I'll plan that for the next PR.

Thanks for kicking off the build again, it passed now.

Reply:
Alert has been added and is also the default.
Code is done, unit tests are done and I've tested with a streamlabs account.
Please review / approve when ready

Reply:
Looks great man!